src/encoded/static/components/filegallery.js

```diff
@@ -3769,8 +3799,8 @@ const CollapsingTitleComponent = ({
 
     const canDownload = downloadEnabled(files);
 
-    // Filter to audits relevant to the current analysis.
-    const filteredAudit = analysis ? filterAuditByPath(audit, analysis['@id']) : {};
+    // Get the audit object relevant to the current analysis
+    const analysisAudit = analysis && analysisAudits ? analysisAudits[analysis['@id']] : null;
```

Comment:
Line 3803 means if **analysis** is _not falsey_ use the result o **analysisAudits ? analysisAudits[analysis['@id']] : null;** else, **analysisAudit** is set to a falsey value. It is the opposite of use **||** on non-booleans outside of if(). That may be confusing to anyone not aware of that Javascript quirk and may think **analysisAudit** would be a boolean.